sktime/classification/deep_learning/base.py

```diff
@@ -119,4 +131,4 @@ def convert_y_to_keras(self, y, label_encoder=None, onehot_encoder=None):
             y = y.reshape(len(y), 1)
             y = onehot_encoder.fit_transform(y)
 
-        return y
+        return y, label_encoder, onehot_encoder
```

Comment:
If you make this a public function, please provide a docstring.

If not meant to be public, please provide a docstring and make it private - three returns are non-obvious why/what.

Reply:
Added in the documentation.

Reply:
Let me know if its okay, or if I need to change the wordings.